src/assets/css/_sass/components/_site-nav.scss

```diff
@@ -20,4 +22,42 @@
       background-color: -color($greys, 20);
     }
   }
+
+  .site-nav__item.nav__item:last-child {
+    .site-nav__link {
+      padding-left: 65px;
+    }
+
+    &::before {
+      content: '';
+      width: 22px;
+      height: 22px;
+      position: absolute;
+      top: 50px;
+      margin-left: 27.5px;
+      background-color: rgba(0,0,0,0.2);
+      box-shadow: 5px 5px 4px rgba(0,0,0,0.1), 5px -5px 4px rgba(0,0,0,0.1), -5px 5px 4px rgba(0,0,0,0.1), -5px -5px 4px rgba(0,0,0,0.1);
+      border-radius: 50%;
+    }
+
+    &::after {
+      content: '';
+      background-image: url("../icons/location.svg");
+      background-size: cover;
+      width: 28px;
+      height: 28px;
+      position: absolute;
+      margin-left: 26px;
+      top: 38px;
+    }
+  }
```

Comment:
Theres a lot going on here, that doesn't quite get the effect right. There are a few ways, this is one, but needs some work and is not the easiest to pull off.
Another way is to inline the svg and do some fancy stuff.
the last way, which is also the easiest is to use the svg filters
eg
```
<filter id="a" width="437.5%" height="325%" x="-168.8%" y="-62.5%" filterUnits="objectBoundingBox">
  <feOffset dy="12" in="SourceAlpha" result="shadowOffsetOuter1"/>
  <feGaussianBlur in="shadowOffsetOuter1" result="shadowBlurOuter1" stdDeviation="7"/>
  <feColorMatrix in="shadowBlurOuter1" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.39209692 0"/>
</filter>
```

They save us a lot of energy when creating the shadow on the icon.

With a new `location-with-shadow.svg` (sorry, had to zip it for github to allow uploading)
[Uploading location-with-shadow.svg.zip…]()

we can replace lines 26-53 with

```
  .site-nav__item:last-child {
    position: relative;

    .site-nav__text::before {
      content: '';
      background-image: url('/assets/icons/location-with-shadow.svg');
      width: 40px;
      height: 40px;
      display: block;
      position: absolute;
      left: 0;
    }
  }
```